packages/wonder-blocks-core/src/util.js

```diff
@@ -33,7 +33,14 @@ export function processStyleList<T: Object>(style?: StyleType<T>) {
     flatten(style).forEach((child: T) => {
         // Check for aphrodite internal property
         if ((child: any)._definition) {
-            stylesheetStyles.push(child);
+            if (
+                typeof process !== "undefined" &&
+                process.env.STORYBOOK_INLINE_APHRODITE
+            ) {
+                inlineStyles.push(child._definition);
```

Comment:
This is good for now.  `aphrodite` really needs some helper methods so that we're not relying on internals indefinitely.

Reply:
Oh nice!  I really like we are showing what has changed hence making the reporting more useful.